<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HttpRequestForm</class>
 <widget class="QWidget" name="HttpRequestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="UrlEdit" name="urlEdit" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="headLayout">
     <item>
      <widget class="QComboBox" name="methodBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>GET</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>POST</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="contentTypeLabel">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Content-Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="contentTypeBox">
       <item>
        <property name="text">
         <string>application/json</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>application/x-www-form-urlencoded </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>multipart/form-data; boundary=</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="headerAddButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="headersTable">
     <column>
      <property name="text">
       <string>键</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="bodyJsonEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UrlEdit</class>
   <extends>QWidget</extends>
   <header>utility.widget.urledit</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
